packages/trader/src/Modules/Account/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -585,8 +586,14 @@ class PersonalDetailsForm extends React.Component {
                                 </FormBody>
                                 <FormFooter>
                                     {status && status.msg && <FormSubmitErrorMessage message={status.msg} />}
+                                    <div className='account-form__footer-note'>
+                                        { localize('Please make sure your information is correct or it may affect your trading experience.') }
+                                    </div>
                                     <Button
-                                        className='account-form__footer-btn btn--primary--default'
+                                        className={classNames('account-form__footer-btn', {
+                                            'btn--primary--default': !is_submit_success,
+                                            'btn--primary--green'  : is_submit_success,
```

Comment:
@mahdiaryayi-fs ahh just one more suggestion: could you make this changes in the `<Button />` component itself? this should be applied to all usages of Button, not only on this page 😄

Reply:
@kelly-binary I checked `<Button />` use cases in deriv trader. We are padding different classNames in different cases. If I add this logic by adding `btn--primary--default` and `btn--primary--green` inside `<Button />` it doesn't work in other places that we are not using primary button and may cause styling conflict.